Monitron.IM_RPC/Command.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace Monitron.ImRpc
+{
+    public class Command
+    {
+        public string Name { get; set; }
+        public List<string> Args { get; set; }
+
+        //parsing the string to method name and args. Acutally I think it can be a ctor instead of a static func (according to design)
+        public static Command Parse(string i_Message)
+        {
+            Command result = null;
+            char[] stringSeparators = new char[] { ' ', '\n', '\t' };
+            List<string> clean = (i_Message.Split(stringSeparators)).ToList();
```

Comment:
Just splitting whitespace works by passing `null` to `split()` but this doesn't handle "long arguemnt"